app/src/main/java/com/nitramite/paketinseuranta/MainMenu.java

```diff
@@ -254,29 +246,14 @@ protected void onCreate(Bundle savedInstanceState) {
         } else {
             PushUtils.unsubscribeFromTopic(PushUtils.TOPIC_UPDATE);
         }
-        /*if (!SP_PARCELS_AUTOMATIC_UPDATE) {
-            if (isMyServiceRunning(ParcelServiceTimer.class)) {
-                stopService(new Intent(MainMenu.this, ParcelServiceTimer.class));
-            }
-        }*/
 
 
         // Intro logic, show intro if not shown yet
-        if (!sharedPreferences.getBoolean(Constants.SP_INTRO_IS_SHOWN, false) && Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
How is this related?

Reply:
Sorry again my ugly habbit, I was looking while doing this pr related thing that there's code which is not valid anymore due min sdk verison. 😭

Reply:
It happens to everyone occasionally. The temptation is too high. :slightly_smiling_face: